webapp/src/projects.tsx

```diff
@@ -480,7 +475,7 @@ export class ProjectsCarousel extends data.Component<ProjectsCarouselProps, Proj
     }
 
     fetchLocalData(): pxt.workspace.Header[] {
-        const headers: pxt.workspace.Header[] = this.getData("header:*")
```

Comment:
Note this was only fetching local headers... but it is called fetchLocal.. is it supposed to be local or cloud+local?